src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -1800,20 +1812,37 @@ public ImmutableArray<DocumentId> GetDocumentIdsWithFilePath(string? filePath)
         private ImmutableDictionary<ProjectId, CompilationTracker> CreateCompilationTrackerMap(ProjectId projectId, ProjectDependencyGraph dependencyGraph)
```

Comment:
Can we rename "projectId" to "projectIdBeingChanged"? Might help with code understanding.

Reply:
Renamed to `changedProjectId` locally; will mark resolved on push

Reply:
➡️ Now pushed.